scripts/generate_changelog.sh

```diff
@@ -27,6 +27,3 @@ docker run --rm -v ${BUNDLES_PATH}:/repository -w /repository -e NEW_RELEASE_TIT
 
```

Comment:
I also propose remove line 25-26 `# generate full changelog`. If we don't push changelog.md file to repository anymore then we don't have to generate it. Also we can remove file `push_changelog.sh` if we don't use it anymore.